### PR TITLE
Elastic search soar

### PR DESCRIPTION
**Pull request by anki-narravula**
Required items, please complete
   
   Change(s):
   - See guidance below

   Reason for Change(s):
   - See guidance below

   Version Updated:
   - Required only for Detections/Analytic Rule templates
   - See guidance below

   Testing Completed:
   - See guidance below

   Checked that the validations are passing and have addressed any issues that are present:
   - See guidance below

# Guidance <- remove section before submitting
-----------------------------------------------------------------------------------------------------------
## Before submitting this PR please ensure that you have read the following sections and filled out the changes, reason for change and testing complete sections:

Thank you for your contribution to the Microsoft Sentinel Github repo.

> Details of the code changes in your submitted PR.  Providing descriptions for pull requests ensures there is context to changes being made and greatly enhances the code review process.  Providing associated Issues that this resolves also easily connects the reason.
   
   Change(s):
   - Updated syntax for XYZ.yaml

   Reason for Change(s):
   - New schema used for XYZ.yaml
   - Resolves ISSUE #1234

   Version updated:
   - Yes
   - Detections/Analytic Rule templates are required to have the version updated

> The code should have been tested in a Microsoft Sentinel environment that does not have any custom parsers, functions or tables, so that you validate no incorrect syntax and execution functions properly.  If your submission requires a custom parser or function, it must be submitted with the PR.

   Testing Completed:
   - Yes/No/Need Help

_Note: If updating a detection, you must update the version field._

> Before the submission has been made, please look at running the KQL and Yaml Validation Checks locally.
> https://github.com/Azure/Azure-Sentinel#run-kql-validation-locally

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes/No/Need Help
   
   _Note: Let us know if you have tried fixing the validation error and need help._

> References: 
> - [Guidance for Detection checks](https://github.com/Azure/Azure-Sentinel#pull-request-detection-template-structure-validation-check)
> - [General contribution guidance](https://github.com/Azure/Azure-Sentinel/wiki#what-can-you-contribute-and-how-can-you-create-contributions)
> - [PR validation troubleshooting](https://github.com/Azure/Azure-Sentinel#pull-request)

-----------------------------------------------------------------------------------------------------------

### CODE REVIEW

File: Solutions/Elastic Search/Playbooks/ElasticSearchPlaybooks/ElasticSearch-EnrichIncident/readme.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+
+![Playbook Designer view](./images/EnrichIndicentElasticSearchWorkflow.png)<br>
+
+### Prerequisites 
+1. Elastic Search Custom Connector needs to be deployed prior to the deployment of this playbook under the same subscription.
+2. API key. To get API Key, login into your Elastic Search instance / Kibana  and navigate to API Keys and generated key.To get Elastic Search API key, follow the instructions in the [documentation](https://www.elastic.co/guide/en/kibana/master/api-keys.html).
+
+### Deployment instructions 
+1. Deploy the playbook by clicking on "Deploy to Azure" button. This will take you to deplyoing an ARM Template wizard.
```
Comment (v-sabiraj) [inline Solutions/Elastic Search/Playbooks/ElasticSearchPlaybooks/ElasticSearch-EnrichIncident/readme.md:17]: Hey @anki-narravula, please fix the typo deplyoing to deploying, thanks.

**Review by v-sabiraj — APPROVED** [review]

**Review by v-dvedak — APPROVED** [review]

**Comment by v-sabiraj** [comment]
Will be working on updating the logo, thanks.

**Review by v-sabiraj — APPROVED** [review]
looks good

**Review by Tichandr — APPROVED** [review]
Looks good

File: Solutions/Elastic Search/Playbooks/ElasticSearchPlaybooks/ElasticSearch-EnrichIncident/readme.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+1.	Click the Azure Sentinel connection resource
+2.	Click edit API connection
+3.	Click Authorize
+4.	Sign in
+5.	Click Save
+6.	Repeat steps for Elastic Search Api  Connection (For authorizing the Elastic Search API connection, API Key needs to be provided)
+#### b. Configurations in Sentinel
+1. In Azure sentinel analytical rules should be configured to trigger an incident with risky user account or host or URL or FileHash or IP Address. 
+2. Configure the automation rules to trigger this playbook
+
```
Comment (manishkumar1991) [inline Solutions/Elastic Search/Playbooks/ElasticSearchPlaybooks/ElasticSearch-EnrichIncident/readme.md:37]: Please add sentinel's responder steps in readme.md file

**Pull request merged**